src/material/list/list-base.ts

```diff
@@ -173,7 +175,7 @@ export abstract class MatListItemBase implements AfterViewInit, OnDestroy, Rippl
     this._hostElement = this._elementRef.nativeElement;
     this._noopAnimations = animationMode === 'NoopAnimations';
 
-    if (!this._listBase._isNonInteractive) {
+    if (_listBase && !_listBase._isNonInteractive) {
```

Comment:
yeah, we chatted about this, but I'm still not sure if this is the best way. If the e.g. list item in a template suddenly ends up being used in e.g. an interactive list, the item will not get that treatment right?

Reply:
It won't, but I think that it'll work more or less like it does in the non-MDC version so that's why I went ahead with the change.

Reply:
Ah if that's the case, then I favor this solution for the simplicity